.github/workflows/wheels.yml

```diff
@@ -178,12 +178,13 @@ jobs:
 
       - name: Store artifacts
         uses: actions/upload-artifact@v3
+        if: ${{success() && github.repository == 'scikit-learn/scikit-learn'}}
```

Comment:
Why add `success()` to the `if` here? I don't know if it was missing in the past or if it was left off on purpose.

Reply:
In case of build or test failure in the previous step, that "Store artifacts" step would not have been executed, right?